src/api/FileVersionAPI.js

```diff
@@ -38,17 +38,26 @@ class FileVersionAPI extends API {
         return `${this.apiHost}/2.0/files/${this.fileId}/annotations`;
     }
 
+    /**
+     * Fetches the annotations for the specified file version id
+     *
+     * @param {string} version - File version ID
+     * @return {Promise} Promise that resolves with annotations
+     */
+    fetchAnnotations(version: string): Promise<any> {
```

Comment:
What are your thoughts on moving the API to BUIE and importing it into box-annotations? I'm wondering if we might be duplicating the API layer in both packages since we'll need to fetch all annotations across versions for the sidebar. Potentially, there could be Cache sharing as well between the packages

Reply:
Yeah, good idea. Once we have it in BUIE, I can delete most of this code.